crates/turbopack-node/js/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "@vercel/turbopack-node",
+  "version": "0.0.0",
+  "description": "turbopack node runtime",
+  "license": "UNLICENSED",
+  "private": true,
+  "scripts": {
+    "check": "tsc --noEmit"
+  },
+  "dependencies": {
+    "loader-runner": "^4.3.0"
```

Comment:
How do we ensure this is installed by the user? Is it one of Next.js' transitive dependencies? Or should we compile it and distribute it with turbopack?

Reply:
Currently the user need to install it to use this feature. That's fine for the first iteration...